### ISSUE TITLE
Не корректная работа кнопки "Поиск" в табличной части документа

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2687]
Originally reported on Google Code with ID 2687

```

    В табличной части документа при поиске необходимого значения при помощи кнопки
"Поиск" помимо необходимых полей есть еще и ненужные поля в данном случае:
- "NUMBER";
- "DOCUMENTDATE".
    Необходимо убрать такие поля.
    В шапочной части документа такого нет.

```

Reported by `NikolayUkleyko` on 2012-02-10 14:25:59

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-02-21 12:19:18
- **Labels added**: Type-Defect, Priority-Medium, GedeminEXE, Usability

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-05-06 12:50:44
- **Status changed**: `Duplicate`
- **Merged into**: #2490
